README.md

```diff
@@ -53,14 +53,15 @@ All additional compatibility is only best effort, or happens to still/already be
 #### Compatibility matrix
 At most 5 kube-state-metrics releases will be recorded below.
 
-| kube-state-metrics | client-go  | **Kubernetes 1.9** | **Kubernetes 1.10** | **Kubernetes 1.11** | **Kubernetes 1.12** |
-|--------------------|------------|--------------------|---------------------|---------------------|---------------------|
-| **v1.2.0**         |  v6.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.3.0**         |  v6.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.3.1**         |  v6.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.4.0**         |  v8.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.5.0**         |  v8.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **master**         |  v8.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
+| kube-state-metrics | client-go  | **Kubernetes 1.10** | **Kubernetes 1.11** | **Kubernetes 1.12** | **Kubernetes 1.14** |
+|--------------------|------------|---------------------|---------------------|---------------------|---------------------|
+| **v1.2.0**         |  v6.0.0    |         ✓           |         ✓           |         ✓           |         -           |
```

Comment:
I believe we'd need to remove one entry from this table as the version support matrix records just 5 releases (as per L54).

We could do either of the two
i) Remove L58. `v1.2.0`
ii) Remove L59. `v1.3.0`

I lean towards removing L59 as `v1.3.1` supersedes `v1.3.0`. Thoughts @mxinden ?

Reply:
@mxinden @tariq1890 Remove v1.2.0 is the right way to do this.

Reply:
After next release, we can both remove v1.3.0 and v1.3.1 as they have no big differences.

Reply:
I'd say let's remove v1.3.1, and maybe even change each of the lines to `v1.3.x` (as we should never bump the client-go version in a patch unless for good reason and then we should have an item in this matrix specific to that). Also am I the only one who feels it's odd that we don't have a Kubernetes 1.13 column?

Reply:
Thanks for catching that @brancz. I think we should definitely add a Kubernetes 1.13 column.